### DISCUSSION TITLE
YouTube video play stopped today suddenly (Windows 10)

### DISCUSSION THREAD

**Discussion by wade-w6** [smplayer-dev/smplayer#650] [General]
SMPlayer has worked fine for YouTube viewing for months, but today it stopped in the middle of a playlist of several videos and now plays nothing from YouTube. It still plays local videos and videos from other streaming services. It seems like somebody at YT threw a switch and changed something. Anybody else experiencing this problem?

**Comment by ThomasSeeker** [comment]
Same here. Also since yesterday.
:-(

**Comment by vasilis74** [comment]
Same here from yesterday. Note, my browser also redirects all links to **m.youtube**.

Ubuntu 20.04.5 LTS (Focal Fossa) 64-bit
Kernel Linux 5.4.0-139-generic x86_64
MATE 1.24.0

SMPlayer © 2006-2021 Ricardo Villalba <ricardo@smplayer.info>
Version: 22.7.0 (revision 10091)
Using Qt 5.12.8 (compiled with Qt 5.12.8)
Using MPV 0.32.0

**Comment by smplayer-dev** [comment]
It's still working for me.

**Comment by wade-w6** [comment]
There was a Windows 10 update that automatically installed Wednesday. I wondered if that had anything to do with it. I updated to the latest version of SMPlayer yesterday (22.7.0) and it didn't help. I'm thinking I'll watch for a Player update in the next week or two.

**Reply by ThomasSeeker** [reply]
> There was a Windows 10 update that automatically installed Wednesday. I wondered if that had anything to do with it. I updated to the latest version of SMPlayer yesterday (22.7.0) and it didn't help. I'm thinking I'll watch for a Player update in the next week or two.

vasilis74 and I, we both run Linux. So it has nothing to do with Windows.

**Comment by smplayer-dev** [comment]
Try updating yt-dlp using the option `Install / Update Youtube support` from the menu `Help`. It should install version 2023.02.17.

**Comment by ThomasSeeker** [comment]
Updating yt-dlp to todays new version 2023.02.17 solves the problem for me.
Thank you Ricardo for the help!

SMPlayer is the best! ;-)

**Comment by wade-w6** [comment]
Yes, the yt-dlp update worked for me, too. Thanks much for the help. I had not noticed there was a separate way to update YouTube support. I'll remember to check that next time.

**Comment by zhouxinghong** [comment]
Also not accessible with latest yt-dlp so far, eg:
https://www.youtube.com/watch?v=zC0NfUoZPbU
Windows 10

**Comment by CoreyPenIT** [comment]
> Try updating yt-dlp using the option `Install / Update Youtube support` from the menu `Help`. It should install version 2023.02.17.

I didn't realise that was an option, thanks.  I think I still prefer my method of creating a symlink in the smplayer directory pointing to a yt-dlp.exe in my system path, where I can update from console with `yt-dlp -U`

This is usally the issue though, for future reference.  yt-dlp has updates fairly often between smplayer releases and it's a good idea to keep it up to date with whatever method you use, for most that will be through smplayer gui.
